Sources/web3swift/Web3/Web3+Instance.swift

```diff
@@ -60,7 +57,7 @@ public class Web3 {
     // FIXME: Rewrite this to CodableTransaction
     public class Personal {
         var provider: Web3Provider
-        //        weak var web3: web3?
+        // FIXME: web3 must be weak
```

Comment:
I guess this will be fixed in another PR?

Reply:
Correct. Just made it more obvious that it needs fixing instead of a commented line of code.
I'll actually update the part after FIXME. Ideally we should remove `web3` reference from here completely.